md2rst.py

```diff
@@ -0,0 +1,195 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+"""
+Implements a markdown to reST converter using the mistune markdown
+parser.
+"""
```

Comment:
Ah, cool.  At first I wondered if you were using pandoc, but this seems simpler.